server/routes/teamEventAPI.js

```diff
@@ -0,0 +1,69 @@
+const express = require("express");
+const db = require("../db/models");
+const auth = require("../middleware/auth");
+
+const router = express.Router();
+
+// 1 - Create a new team event type
+router.post("/api/team-event", auth, (req, res) => {
+  if (req.body.duration) {
+    //
+    db.EventType.create({ ...req.body, teamId: req.teamId })
```

Comment:
@WhoaNellie Have you added teamId to the EventType model in your PR? If not then Matt should add it in this one.

Also wouldn't the teamId be a part of the req.body?

Reply:
I did add it yes